boards/nrf52840_mdk_dfu/src/io.rs

```diff
@@ -0,0 +1,65 @@
+use core::fmt::Write;
```

Comment:
What's specific about this file that doesn't exist in one of the `third_party/tock/boards/nordic/foo/src/io.rs`? We shouldn't be duplicating code.

If the change is a one-liner over the `nrf52840_dongle` board, we should keep it as a patch applied by `./setup.sh`.

Same question for the `main.rs`.

Reply:
It's a different board. Buttons and LEDs are different.

Reply:
Resolving for now. Ideally we would just have some diff patch to apply, but I don't want to block this pull-request for that.